gazebo/models/assignment_yarp-find-rgb-world-interface/model.urdf

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
+<!-- <?xml version="1.0" ?> -->
+<robot name="assignment_yarp-find-rgb-world-interface">
+
+  <link name="base_link"/>
+
+  <joint name="fake_base_fixed_joint" type="fixed">
+    <parent link="base_link"/>
+    <child link="base"/>
+  </joint>
+  
+  <link name="base">
+    <collision>
```

Comment:
Actually, you need neither the collision nor the inertial parts.

Reply:
In theory not, but I don't remember why in practice you need a phisycal object(a cube) for using this plugin.

@traversaro do you know why?

BTW I set it very far, it should not pop out in the world